### PR TITLE
Fixing the new restructuring causing break in the build of library an…

### PR DESCRIPTION
**Pull request by mehdi-goli**
…d breaking DPCPP compiler.

### CODE REVIEW

**Comment by mehdi-goli** [comment]
@Bensuo  the git action fails as it cannot clone any repo except sycl-blas Codeplay software. Maybe a bug in git action?

File: common/include/common/common_utils.hpp [hunk 1]
```diff
@@ -17,9 +17,8 @@
 #include <vector>
 
 #include "benchmark_cli_args.hpp"
-
-#include <common/vector_utils.hpp>
-#include <common/clwrap.hpp>
+#include "blas_meta.h"
+//#include <common/clwrap.hpp>
```
Comment (Bensuo) [inline common/include/common/common_utils.hpp:21]: Can we remove this comment?

File: common/include/common/common_utils.hpp [hunk 2]
```diff
@@ -587,29 +586,28 @@ static inline void warmup(function_t func, args_t&&... args) {
...
 // Declared here, defined separately in the specific utils.hpp files
 
 /**
  * @fn time_events
  * @brief Times n events, and returns the aggregate time.
  */
-template <typename event_t>
-static inline cl_ulong time_events(std::vector<event_t> es) {
-  cl_ulong total_time = 0;
+template < typename event_t>
```
Comment (Bensuo) [inline common/include/common/common_utils.hpp:596]: This formatting seems a little weird, has clang-format been run on this file? If not it probably should.

File: include/blas_meta.h [hunk 3]
```diff
@@ -136,6 +135,30 @@ static SYCL_BLAS_INLINE index_t roundUp(index_t x, index_t y) {
...
+int append_vector(vector_t &lhs_vector, vector_t const &rhs_vector) {
+  lhs_vector.insert(lhs_vector.end(), rhs_vector.begin(), rhs_vector.end());
+  return 0;
+}
+
+template <typename first_vector_t, typename... other_vector_t>
+first_vector_t concatenate_vectors(first_vector_t first_vector,
+                                   other_vector_t &&... other_vectors) {
+  int first_Vector_size = static_cast<int>(first_vector.size());
+  int s[] = {vec_total_size(first_Vector_size, other_vectors)..., 0};
```
Comment (Bensuo) [inline include/blas_meta.h:154]: I realise this is not really relevant to this PR, but why do `vec_total_size`/`append_vector` return 0 and get assigned to arrays that are never used?

**Review by Bensuo — COMMENTED** [review]
Looks good to me, just a couple of minor comments.

**Comment by Bensuo** [comment]
Reviewed the wrong one, my mistake!

**Pull request closed without merging**